### PR TITLE
while sign out, the imported wallets should be able show on UI 

### PR DESCRIPTION
**Pull request by rueshyna**
- show sidebar when signing out
- import wallet shouldn't require log in

### CODE REVIEW

File: pages/import-wallet.tsx [hunk 1]
```diff
@@ -18,33 +18,7 @@ function Import() {
...
-        router.replace("/");
-      }
-    };
-    (async () => {
-      if (!state?.address && state?.beaconWallet) {
-        await connectWallet();
-      }
-    })();
-  }, [state, dispatch, router]);
+  useEffect(() => {}, [state, dispatch, router]);
```
Comment (JulesGuesnon) [inline pages/import-wallet.tsx:21]: Why everything has been deleted?

**Review by JulesGuesnon — COMMENTED** [review]
So overall looks good, but while testing I had another deletion issue. The contracts were deleted, but there was still aliases, so I'll try to push a fix in this PR

File: pages/import-wallet.tsx (hunk 1 above)
Comment (rueshyna) [reply pages/import-wallet.tsx:21]: In the step of importing wallet, no needed for making Tezos transactions so the login action isn't actually required.

**Comment by JulesGuesnon** [comment]
Actually, I think I just didn't paid attention and thought an alias was a contract but it was a user alias. I fixed small UI padding issue tho

**Review by JulesGuesnon — APPROVED** [review]

**Pull request merged**